tool/src/org/antlr/v5/tool/ErrorType.java

```diff
@@ -417,7 +417,7 @@ public enum ErrorType {
 	 * The following sets of rules are mutually left-recursive
 	 * <em>[rules]</em></p>
 	 */
-	LEFT_RECURSION_CYCLES(119, "The following sets of rules are mutually left-recursive <arg:{c| [<c:{r|<r.name>}; separator=\", \">]}; separator=\" and \">", ErrorSeverity.ERROR),
+	LEFT_RECURSION_CYCLES(119, "The following set of rules are mutually left-recursive [<arg:{r|<r.name>}; separator=\", \">]", ErrorSeverity.ERROR),
```

Comment:
Did you mean "the following rules" ? Please also update the comment ?

Reply:
Originally it was `sets` that's why I didn't change original semantics. But I can rephrase with `the following rules`, yes, probably it looks a bit more clear for an end user.